documentation/shift_view_POST.md

```diff
@@ -0,0 +1,128 @@
+# Master Schedule Shift View
+
+Returns json data about shift assignment
+
+- **URL:**
+
+  /api/shift_view
+
+- **Method:**
+
+  `POST`
+
+- **URL Params:**
+
+  None
+
+- **Data Params:**
+
+  [
+    {
+      timeBlockId: 1,
+      userId: 1
+    },
+    {
+
+    }
+  ]
```

Comment:
Not sure if this is possible to clean up or not, but as is it is kind of hard to read